File: librarymanagementsystem-backend/src/librarymanagementsystembackend/Application/Features/MaterialCopies/Commands/Create/CreatedMaterialCopyResponse.cs
using NArchitecture.Core.Application.Responses;

namespace Application.Features.MaterialCopies.Commands.Create;

public class CreatedMaterialCopyResponse : IResponse
{
    public Guid Id { get; set; }
    public DateTime DateReceipt { get; set; }
    public string Status { get; set; }
    public Guid MaterialId { get; set; }
    public Guid BranchId { get; set; }
    public Guid LocationId { get; set; }
}

File: librarymanagementsystem-frontend/src/app/features/models/responses/material-copies/created-material-copy-response.ts
export interface CreatedMaterialCopyResponse {
	id: string;
	dateReceipt: string;
	status: string;
	isReserved: boolean;
	isReservable: boolean;
	materialId: string;
	branchId: string;
	locationId: string;
    createdDate: Date;
}